scripts/build-run-test-dockerfile.sh

```diff
@@ -28,6 +28,49 @@ SERVICE_CONTROLLER_SOURCE_PATH=${SERVICE_CONTROLLER_SOURCE_PATH:-$DEFAULT_SERVIC
 SERVICE_CONTROLLER_E2E_TEST_PATH="${SERVICE_CONTROLLER_SOURCE_PATH}/test/e2e"
 
 KUBECONFIG_LOCATION="${KUBECONFIG:-"$HOME/.kube/config"}"
+AWS_CREDS_FILE_LOCATION="$HOME/.aws/credentials"
+
+# create an empty web-identity-token file inside Docker build context for
+# the test container.
+# If this script is running as part of prowjob then we will copy web-identity-token
+# in this file to be used inside test container
+touch "$SCRIPTS_DIR"/web-identity-token >/dev/null
+
+# path of web-identity-token in test container
+TEST_CONTAINER_WEB_IDENTITY_TOKEN_FILE="/root/web-identity-token"
+# run python tests using new AWS profile to allow credential auto rotation
+ACK_TEST_AWS_PROFILE="ack-test"
+# If 'AWS_PROFILE' variable is set, use it as source profile for 'ack-test'
+# profile. Use 'default' as fallback
+ACK_TEST_SOURCE_AWS_PROFILE=${AWS_PROFILE:-"default"}
+# location of new credential file which will be copied in test container
+# Keep this file in same location as '~/.aws/credentials' file
+TEST_AWS_CREDS_FILE_LOCATION="$HOME/.aws/ack-test-credentials"
```

Comment:
Yes, this file is supposed to be on the system launching script so it can be mounted during docker build.
I kept it in same place as ~/.aws/credentials file because i do not want the creds file to go in tmp directory(Just being careful)